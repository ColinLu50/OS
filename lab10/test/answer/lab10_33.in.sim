20026
873
1074
645
1050
632
