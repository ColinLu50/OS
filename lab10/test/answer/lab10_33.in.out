FCFS
423 -> 377 -> 514 -> 243 -> 571 -> 219 -> 254 -> 430 -> 120 -> 297 -> 570 -> 452 -> 113 -> 229 -> 373 -> 201 -> 541 -> 635 -> 513 -> 586 -> 154 -> 21 -> 333 -> 612 -> 38 -> 391 -> 110 -> 127 -> 557 -> 141 -> 223 -> 467 -> 138 -> 574 -> 59 -> 186 -> 32 -> 27 -> 137 -> 584 -> 307 -> 60 -> 624 -> 463 -> 459 -> 546 -> 568 -> 291 -> 403 -> 420 -> 647 -> 644 -> 366 -> 220 -> 219 -> 42 -> 529 -> 224 -> 40 -> 502 -> 374 -> 327 -> 607 -> 573 -> 326 -> 528 -> 178 -> 521 -> 616 -> 517 -> 34 -> 386 -> 544 -> 12 -> 337 -> 345 -> 46 -> 384 -> 279 -> 503 -> 599 -> 86 -> 323 -> 616 -> 183 -> 651
20026
SSTF
423 -> 420 -> 430 -> 452 -> 459 -> 463 -> 467 -> 502 -> 503 -> 513 -> 514 -> 517 -> 521 -> 528 -> 529 -> 541 -> 544 -> 546 -> 557 -> 568 -> 570 -> 571 -> 573 -> 574 -> 584 -> 586 -> 599 -> 607 -> 612 -> 616 -> 616 -> 624 -> 635 -> 644 -> 647 -> 651 -> 403 -> 391 -> 386 -> 384 -> 377 -> 374 -> 373 -> 366 -> 345 -> 337 -> 333 -> 327 -> 326 -> 323 -> 307 -> 297 -> 291 -> 279 -> 254 -> 243 -> 229 -> 224 -> 223 -> 220 -> 219 -> 219 -> 201 -> 186 -> 183 -> 178 -> 154 -> 141 -> 138 -> 137 -> 127 -> 120 -> 113 -> 110 -> 86 -> 60 -> 59 -> 46 -> 42 -> 40 -> 38 -> 34 -> 32 -> 27 -> 21 -> 12
873
SCAN
423 -> 420 -> 403 -> 391 -> 386 -> 384 -> 377 -> 374 -> 373 -> 366 -> 345 -> 337 -> 333 -> 327 -> 326 -> 323 -> 307 -> 297 -> 291 -> 279 -> 254 -> 243 -> 229 -> 224 -> 223 -> 220 -> 219 -> 201 -> 186 -> 183 -> 178 -> 154 -> 141 -> 138 -> 137 -> 127 -> 120 -> 113 -> 110 -> 86 -> 60 -> 59 -> 46 -> 42 -> 40 -> 38 -> 34 -> 32 -> 27 -> 21 -> 12 -> 0 -> 430 -> 452 -> 459 -> 463 -> 467 -> 502 -> 503 -> 513 -> 514 -> 517 -> 521 -> 528 -> 529 -> 541 -> 544 -> 546 -> 557 -> 568 -> 570 -> 571 -> 573 -> 574 -> 584 -> 586 -> 599 -> 607 -> 612 -> 616 -> 624 -> 635 -> 644 -> 647 -> 651
1074
C-SCAN
423 -> 420 -> 403 -> 391 -> 386 -> 384 -> 377 -> 374 -> 373 -> 366 -> 345 -> 337 -> 333 -> 327 -> 326 -> 323 -> 307 -> 297 -> 291 -> 279 -> 254 -> 243 -> 229 -> 224 -> 223 -> 220 -> 219 -> 201 -> 186 -> 183 -> 178 -> 154 -> 141 -> 138 -> 137 -> 127 -> 120 -> 113 -> 110 -> 86 -> 60 -> 59 -> 46 -> 42 -> 40 -> 38 -> 34 -> 32 -> 27 -> 21 -> 12 -> 0 -> 652 -> 651 -> 647 -> 644 -> 635 -> 624 -> 616 -> 612 -> 607 -> 599 -> 586 -> 584 -> 574 -> 573 -> 571 -> 570 -> 568 -> 557 -> 546 -> 544 -> 541 -> 529 -> 528 -> 521 -> 517 -> 514 -> 513 -> 503 -> 502 -> 467 -> 463 -> 459 -> 452 -> 430
645
LOOK
423 -> 420 -> 403 -> 391 -> 386 -> 384 -> 377 -> 374 -> 373 -> 366 -> 345 -> 337 -> 333 -> 327 -> 326 -> 323 -> 307 -> 297 -> 291 -> 279 -> 254 -> 243 -> 229 -> 224 -> 223 -> 220 -> 219 -> 201 -> 186 -> 183 -> 178 -> 154 -> 141 -> 138 -> 137 -> 127 -> 120 -> 113 -> 110 -> 86 -> 60 -> 59 -> 46 -> 42 -> 40 -> 38 -> 34 -> 32 -> 27 -> 21 -> 12 -> 430 -> 452 -> 459 -> 463 -> 467 -> 502 -> 503 -> 513 -> 514 -> 517 -> 521 -> 528 -> 529 -> 541 -> 544 -> 546 -> 557 -> 568 -> 570 -> 571 -> 573 -> 574 -> 584 -> 586 -> 599 -> 607 -> 612 -> 616 -> 624 -> 635 -> 644 -> 647 -> 651
1050
C-LOOK
423 -> 420 -> 403 -> 391 -> 386 -> 384 -> 377 -> 374 -> 373 -> 366 -> 345 -> 337 -> 333 -> 327 -> 326 -> 323 -> 307 -> 297 -> 291 -> 279 -> 254 -> 243 -> 229 -> 224 -> 223 -> 220 -> 219 -> 201 -> 186 -> 183 -> 178 -> 154 -> 141 -> 138 -> 137 -> 127 -> 120 -> 113 -> 110 -> 86 -> 60 -> 59 -> 46 -> 42 -> 40 -> 38 -> 34 -> 32 -> 27 -> 21 -> 12 -> 651 -> 647 -> 644 -> 635 -> 624 -> 616 -> 612 -> 607 -> 599 -> 586 -> 584 -> 574 -> 573 -> 571 -> 570 -> 568 -> 557 -> 546 -> 544 -> 541 -> 529 -> 528 -> 521 -> 517 -> 514 -> 513 -> 503 -> 502 -> 467 -> 463 -> 459 -> 452 -> 430
632
